src/utils.jl

```diff
@@ -192,7 +192,7 @@ end
 function read_json_str(json)
     # length check is to ensure that isfile doesn't thrown an error
     # see issue for details https://github.com/JuliaLang/julia/issues/39774
-    !(json isa VectorString) && length(json) < 255 && isfile(json) ?
           VectorString(Mmap.mmap(json)) : 
           json
+    !(json isa VectorString) && sizeof(json) < 255 && isfile(json) ?
```

Comment:
Ummm, this still needs to be up on line 195, right?

Reply:
Whoops, unintentional, probably from staging just some lines from the diff. One moment.